docs/php/activities.md

```diff
@@ -314,7 +314,7 @@ $activityClient->completeExceptionallyByToken($taskToken, new \Error("activity f
 
 ## Local Activity
 
-In Temporal there is a concept of [Local Activity](/docs/concepts/what-is-a-local-activity). To create a local 
+In Temporal there is a concept of [Local Activity](/docs/concepts/what-is-a-local-activity). To create a local
```

Comment:
Get rid of the first sentence, and hyperlink 'Local Activity' in the next sentence.